src/lib.rs

```diff
@@ -128,12 +128,18 @@ enum Fields {
 
 /// Defines a struct field.
 #[derive(Debug, Clone)]
-struct Field {
+pub struct Field {
     /// Field name
     name: String,
 
     /// Field type
     ty: Type,
+
+    /// Field documentation
+    documentation: String,
```

Comment:
Documentation strings are tricky. Reading the diff, it looks like only a single doc string line is supported. I'm not sure what the best option would be to support multi lines. Either "\n" could be detected in the documentation string and then it all gets formatted, or `documentation` could be a `Vec<String>`.

Thoughts"

Reply:
How about two different functions that do each? Have `doc()` take a `Vec<String>` to stay the same as `annotation()` and something like `doc_with_newlines()`.